frontend/src/components/Column.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useRef } from 'react';
+import Card from './Card';
+
+const Column = ({ title }) => {
+  const draggingItem = useRef();
+  const dragOverItem = useRef();
+
+  const [list, setList] = useState(['1', '2s', '3', '4']);
```

Comment:
is `2s` a typo?

Reply:
It was a typo, however this part is changed